server/views/pages/prisons/sessionTemplates/editSingleSessionTemplate.njk

```diff
@@ -54,7 +54,7 @@
       {% set infoBannerHtml %}
       <p data-test="total-booked-test">This visit session template currently has {{ visitStats.visitCount }} future {{ 'visits' if visitStats.visitCount >= 2 else 'visit' }} booked</p>
       {% for stats in visitStats.visitsByDate %}
-        <p>For the date: {{ stats.visitDate | formatDate }}, there is currently {{ stats.visitCount }} {{ 'visits' if stats.visitCount >= 2 else 'visit' }} booked</p>
+        <p>For the date: {{ stats.visitDate | formatDate }}, there is currently {{ stats.visitCounts.open }} open {{ 'visits' if stats.visitCounts.open >= 2 else 'visit' }} and {{ stats.visitCounts.closed }} closed {{ 'visits' if stats.visitCounts.closed >= 2 else 'visit' }} booked</p>
```

Comment:
This works fine (and I see it's already merged) but for future reference don't forget we've got the `pluralise` Nunjucks filter! https://github.com/ministryofjustice/hmpps-visits-internal-admin-ui/blob/a6f7ff6b1667ff85ff26e1988cfd06e15309174c/server/utils/nunjucksSetup.test.ts#L150